dmt-custom-adminrole-setup.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
The script names doesn't imply correct message about what it is for.
Renaming it to something like dmt-admin-user-setup.sh and dmt-admin-roles-setup.sh might help. similarly for user script.

Reply:
Sure, I would change it to then,

dmt-admin-setup.sh -> dmt-admin-user-setup.sh
dmt-custom-adminrole-setup.sh -> dmt-admin-roles-setup.sh
dmt-custom-userrole-setup.sh -> dmt-user-roles-setup.sh
dmt-user-setup.sh --> dmt-user-setup.sh

Hope this will work and let me know or comment in case of any modification required in above rename file mapping..

Reply:
Done